kokoro/scripts/test/go_test.sh

```diff
@@ -78,7 +78,16 @@ GO_VERSION="1.19"
 
 # Download and install a newer version of go.
 # Install from a GCS bucket to avoid being throttled by go.dev.
-gsutil cp "gs://stackdriver-test-143416-go-install/go${GO_VERSION}.linux-amd64.tar.gz" - | \
+unset ARCH
+case "$(uname -m)" in
+  "x86_64")
+    ARCH="amd64"
+    ;;
+  "aarch64")
+    ARCH="arm64"
+    ;;
+esac
+gsutil cp "gs://stackdriver-test-143416-go-install/go${GO_VERSION}.linux-${ARCH}.tar.gz" - | \
```

Comment:
this tar.gz doesn't exist for ARM yet, someone needs to upload one for ARM.

Reply:
Already did (for 1.19 at least). (Would this be a good opportunity to upgrade-to-latest or do we typically leave that for when things break and we're forced to upgrade?)

Reply:
there's a known issue with 1.20 and the library we're using to turn test output into XML. Unclear if we're stiill affected now that we turned off json though. needs investigation.